v1/content.yaml

```diff
@@ -200,6 +200,78 @@ paths:
                 page: '{{page}}'
       x-monitor: false
 
+  /title/{title}/{revision}:
```

Comment:
Why are we adding a completely separate endpoint instead of just adding and optional `{/revision}` parameter to the existing one?